### PR TITLE
Allow users to disable plugins via config

### PR DESCRIPTION
**Pull request by zenweasel**
Resolves #NA
Impact: **minor**  
Type: **feature**

## Issue
There is no good and easy way to disable a plugin through configuration

## Solution
Keep a list of disabled plugins and don't load/import any of those

## Breaking changes
None

## Testing
1. Start Reaction and observe no problems
1. Edit `.reaction/pluginConfigs` and add `search-mongo` to the array for `disabledPlugins`.
1. Restart Reaction
1. Observe that the mongo search is disabled

### CODE REVIEW

File: .reaction/pluginConfig.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const disabledPlugins = [];
```
Comment (aldeed) [inline .reaction/pluginConfig.js]: Can you add a comment above here explaining what goes in the array? At first glance I would have thought the plugin's name, but it looks like actually the directory name.

Also, should we gitignore this file? (after this initial commit)

**Comment by zenweasel** [comment]
@aldeed I wonder if sort of tangential to this if we should move all the collection creation stuff for `search-mongo` so that those Collections are never created if the plugin is disabled?

**Comment by aldeed** [comment]
@zenweasel Yes as a general rule we need to get all collection creation moved into the related plugin startup. It would be done soon as part of https://github.com/reactioncommerce/reaction/issues/4996 if not now

**Comment by zenweasel** [comment]
@aldeed Requested changes made

**Pull request merged**